torchvision/models/detection/rpn.py

```diff
@@ -17,8 +17,7 @@
 @torch.jit.unused
 def _onnx_get_num_anchors_and_pre_nms_top_n(ob, orig_pre_nms_top_n):
     # type: (Tensor, int) -> Tuple[int, int]
-    from torch.onnx import operators
-    num_anchors = operators.shape_as_tensor(ob)[1].unsqueeze(0)
+    num_anchors = ob.shape[1].unsqueeze(0)
```

Comment:
So now we don't need to use `operators.shape_as_tensor`? Does ONNX now trace the `.shape` values from tensors and keep them as dynamic variables?

Reply:
Yes, now we are able to trace the shapes dynamically.